psst/case/descriptors.py

```diff
@@ -96,6 +96,10 @@ class Bus(Descriptor):
     name = 'bus'
     ty = pd.DataFrame
 
+class BusString(Descriptor):
```

Comment:
BusString used here

Reply:
The `BusString` class is defined here but no field is set as the `BusString` descriptor.

https://github.com/ames-market/psst/blob/817c2d15443e2ef901ba56aa7607bcbc0e654663/psst/case/__init__.py#L25-L38

Somewhere in the above lines, you'll need to add `bus_string = BusString()`

Reply:
whoops. thanks!